libs/shared/contentful/src/lib/factories.ts

```diff
@@ -23,6 +29,65 @@ import {
 } from './queries/services-with-capabilities';
 import { ContentfulErrorResponse } from './types';
 
+export const CapabilityServiceByPlanIdsQueryFactory = (
+  override?: Partial<CapabilityServiceByPlanIdsQuery>
+): CapabilityServiceByPlanIdsQuery => {
+  return {
+    purchaseCollection: {
+      items: [
+        {
+          stripePlanChoices: [faker.string.sample()],
+          offering: {
+            capabilitiesCollection: {
+              items: [
+                {
+                  slug: faker.string.sample(),
+                  servicesCollection: {
+                    items: [
+                      {
+                        oauthClientId: faker.string.sample(),
+                      },
+                    ],
+                  },
+                },
+              ],
+            },
+          },
+        },
+      ],
+    },
+    ...override,
+  };
+};
+
+export const CapabilityOfferingResultFactory = (
+  override?: Partial<CapabilityOfferingResult>,
+  capabilitiesCollection?: CapabilityCapabilitiesResult[]
+): CapabilityOfferingResult => ({
+  capabilitiesCollection: {
+    items: [...(capabilitiesCollection ?? [])],
+  },
+  ...override,
+});
+
+export const CapabilityCapabilitiesResultFactory = (
+  override?: Partial<CapabilityCapabilitiesResult>,
+  servicesCollection?: CapabilityServicesResult[]
```

Comment:
Each factory should only ever take a single parameter `override`

Reply:
Thank you! As discussed on Slack, updated and moved to its own file. Please let me know what further improvements I need to do.

Reply:
Update for future ref: comments below regarding improvements 🙂